ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/TaskService.cs

```diff
@@ -102,5 +102,20 @@ public Task<bool> ExistsAsync(Guid id, CancellationToken cancellationToken)
             cancellationToken.ThrowIfCancellationRequested();
             return _repository.ExistsAsync(id, cancellationToken);
         }
+
+        public async Task<List<string>> GetHoursByDatesAsync(GetHoursDTO hoursDto, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            var hourDictionary = await _repository.GetHoursByDatesAsync(hoursDto.Date, hoursDto.EmployeeId, cancellationToken);
+            List<string> list = new List<string>();
+
+            foreach (var entry in hourDictionary)
+            {
+                list.Add(entry.Key + ": " + entry.Value);
```

Comment:
try to use string.Join for combining results, and every new day should be from new line